src/helics/shared_api_library/backup/helics_api.h

```diff
@@ -193,41 +196,41 @@ const HelicsBool HELICS_TRUE = 1;
 const HelicsBool HELICS_FALSE = 0;
 
 typedef enum {
-    HELICS_ITERATION_REQUEST_NO_ITERATION,
-    HELICS_ITERATION_REQUEST_FORCE_ITERATION,
-    HELICS_ITERATION_REQUEST_ITERATE_IF_NEEDED
+   HELICS_ITERATION_REQUEST_NO_ITERATION,
+   HELICS_ITERATION_REQUEST_FORCE_ITERATION,
+   HELICS_ITERATION_REQUEST_ITERATE_IF_NEEDED
 } HelicsIterationRequest;
 
 typedef enum {
-    HELICS_ITERATION_RESULT_NEXT_STEP,
-    HELICS_ITERATION_RESULT_ERROR,
-    HELICS_ITERATION_RESULT_HALTED,
-    HELICS_ITERATION_RESULT_ITERATING
+   HELICS_ITERATION_RESULT_NEXT_STEP,
+   HELICS_ITERATION_RESULT_ERROR,
+   HELICS_ITERATION_RESULT_HALTED,
+   HELICS_ITERATION_RESULT_ITERATING
 } HelicsIterationResult;
 
 typedef enum {
-    HELICS_STATE_STARTUP = 0,
-    HELICS_STATE_INITIALIZATION,
-    HELICS_STATE_EXECUTION,
-    HELICS_STATE_FINALIZE,
-    HELICS_STATE_ERROR,
-
-    HELICS_STATE_PENDING_INIT,
-    HELICS_STATE_PENDING_EXEC,
-    HELICS_STATE_PENDING_TIME,
-    HELICS_STATE_PENDING_ITERATIVE_TIME,
-    HELICS_STATE_PENDING_FINALIZE,
-    HELICS_STATE_FINISHED
+   HELICS_STATE_STARTUP = 0,
+   HELICS_STATE_INITIALIZATION,
+   HELICS_STATE_EXECUTION,
+   HELICS_STATE_FINALIZE,
+   HELICS_STATE_ERROR,
+
+   HELICS_STATE_PENDING_INIT,
+   HELICS_STATE_PENDING_EXEC,
+   HELICS_STATE_PENDING_TIME,
+   HELICS_STATE_PENDING_ITERATIVE_TIME,
+   HELICS_STATE_PENDING_FINALIZE,
+   HELICS_STATE_FINISHED
 } HelicsFederateState;
 
 typedef struct HelicsComplex {
-    double real;
-    double imag;
+   double real;
+   double imag;
 } HelicsComplex;
 
 typedef struct HelicsError {
-    int32_t errorCode;
-    const char* message;
+   int32_t errorCode;
```

Comment:
This `errorCode` still hasn't been changed?

Reply:
updated the file without regenerating it first,  should be updated now